OWNERS

```diff
@@ -0,0 +1,13 @@
+component: "Monitoring"
+
+reviewers:
+- LiliC
+- paulfantom
+- s-urbaniak
+- simonpasquier
```

Comment:
we should add @dgrisonnet too

Reply:
we can't yet because he's not part of the OpenShift org yet